.github/workflows/build.yml

```diff
@@ -38,3 +38,33 @@ jobs:
 
       - name: Test
         run: npm test
+
+  coverage:
+    name: Test Coverage - Node.js 17.x
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 17
+        uses: actions/setup-node@v2
+        with:
+          node-version: 17
+          cache: npm
+
+      - name: Install
+        run: npm install
+
+      - name: Compile
+        run: npm run build
+
+      - name: Lint
+        run: npm run lint
+
+      - name: Test
+        run: npm test
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
The token doesn't appear to be needed for public repos.

Reply:
Good point :+1: